node/typings.d.ts

```diff
@@ -163,6 +163,15 @@ interface Organization {
   sellers?: Seller[]
 }
 
+export type GetOrganizationsByEmailWithStatus = {
+  costId: string
+  orgId: string
+  roleId: string
+  id: string
+  clId: string
+  status: string
+}
+
```

Comment:
I'm not sure this should be in this file... or should it be an `interface` like the other cases here?

Reply:
done